### ISSUE TITLE
Dropdown Menus (Don't work without JS)

### ISSUE THREAD

**Issue by kodecount** [wet-boew/wet-boew#7847]
Is their anyone currently working on a the dropdown menus for when no JS or wbdisable mode is turned on?

CSS Dropdown Menus (???):
https://www.justsoftwaresolutions.co.uk/articles/css_menus.html
http://stackoverflow.com/questions/10468554/dropdown-menu-without-javascript
https://deliciousthemes.com/create-a-drop-down-navigation-menu-with-html5-and-css3/

**Comment by LaurentGoderre** [comment]
For accessibility reasons, dropdown menu need JavaScript enabled. That's why when JavaScript is disabled, the top level link in the menu bring you to a landing page that serves the same functionality as the dropdown menus

**Comment by kodecount** [comment]
Well I can't seem to find this magic list in wbdisable or with JS off the menu is completely useless.

**Comment by LaurentGoderre** [comment]
Hmmmm...that is what we said needed to be done all along but it doesn't seem to be implemented many places....but it should

**Comment by kodecount** [comment]
Another example is https://www.canada.ca/en.html?=undefined&wbdisable=true the menu links do nothing.

**Comment by kodecount** [comment]
Another example is https://www.canada.ca/en.html?=undefined&wbdisable=true the menu links do nothing.

@MaximMartel

**Comment by duboisp** [comment]
I agree with @LaurentGoderre about when the mega menu do not work, the top level menu item should point to a landing page that serve the same functionality. I confirm that was the expected design pattern. I didn't look on how that was documented.

Task: 
* [ ] Review how that design pattern is documented, if documented.
* [ ] Change globally in wet-boew [the default top menu](https://github.com/wet-boew/wet-boew/blob/master/site/includes/sitemenucontent.hbs) in a way it support that design pattern and create new landing page if needed.
* [ ] Assign someone to do the above task

@kodecount thanks for reporting this issue, regarding "canada.ca" please open a [new issue on GCweb](https://github.com/wet-boew/GCWeb/issues/new).

**Comment by kodecount** [comment]
I didn't look into if documentation exist either. Maybe a README_ACCESSIBILITY_IMPLEMENTATION.md or something along that line should be in WET folder when you download and have quick guide information and/or links to areas a developer should read up on before using WET.
